app/models/server/models/_Base.d.ts

```diff
@@ -0,0 +1,16 @@
+export declare class Base {
+	model: any;
+
+	constructor(nameOrModel: string);
+
+	tryEnsureIndex(...args: any[]): any;
+
+	findOne(...args: any[]): any;
+	static findOne(...args: any[]): any;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class method 'findOne' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/RocketChat/Rocket-Chat/pullRequest?prid=4420492)